tests/Unit/PointwiseFunctions/Xcts/Test_CenterOfMass.cpp

```diff
@@ -99,55 +106,156 @@ void test_center_of_mass_surface_integral(const double distance) {
       // Get required fields on the interface
       const auto shifted_fields = solution.variables(
           shifted_coords,
-          tmpl::list<
-              Xcts::Tags::ConformalFactor<DataVector>,
-              Xcts::Tags::ConformalMetric<DataVector, 3, Frame::Inertial>,
-              Xcts::Tags::InverseConformalMetric<DataVector, 3,
-                                                 Frame::Inertial>>{});
+          tmpl::list<Xcts::Tags::ConformalFactor<DataVector>>{});
       const auto& conformal_factor =
           get<Xcts::Tags::ConformalFactor<DataVector>>(shifted_fields);
-      const auto& conformal_metric =
-          get<Xcts::Tags::ConformalMetric<DataVector, 3, Frame::Inertial>>(
-              shifted_fields);
-      const auto& inv_conformal_metric = get<
-          Xcts::Tags::InverseConformalMetric<DataVector, 3, Frame::Inertial>>(
-          shifted_fields);
-
-      // Compute outward-pointing unit normal.
-      const auto conformal_r = magnitude(inertial_coords, conformal_metric);
-      const auto conformal_unit_normal =
-          tenex::evaluate<ti::I>(inertial_coords(ti::I) / conformal_r());
 
       // Compute area element
-      const auto sqrt_det_conformal_metric =
-          Scalar<DataVector>(sqrt(get(determinant(conformal_metric))));
-      const auto conformal_area_element =
-          area_element(inv_jacobian, boundary_direction, inv_conformal_metric,
-                       sqrt_det_conformal_metric);
+      const auto flat_area_element =
+          euclidean_area_element(inv_jacobian, boundary_direction);
 
-      // Integrate
+      // Evaluate surface integral
       const auto surface_integrand = Xcts::center_of_mass_surface_integrand(
-          conformal_factor, conformal_unit_normal);
+          conformal_factor, inertial_coords);
       for (int I = 0; I < 3; I++) {
         surface_integral.get(I) += definite_integral(
-            surface_integrand.get(I) * get(conformal_area_element), face_mesh);
+            surface_integrand.get(I) * get(flat_area_element), face_mesh);
       }
     }
   }
 
   // Check result
-  auto custom_approx = Approx::custom().epsilon(10. / distance).scale(1.0);
+  auto custom_approx = Approx::custom().epsilon(TOLERANCE).scale(1.0);
```

Comment:
Can you keep this at O(1/distance)?

Reply:
It doesn't work for this test because the CoM integral diverges as we increase R. I believe this happens due to round-off errors when canceling larger and larger terms. I describe a bit of this at the top of this file.